.github/pull_request_template.md

```diff
@@ -8,6 +8,7 @@ Please include a description of this change and which issue it fixes. If no issu
 * [ ] Potential edge cases thought about (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
 * [ ] Usability review done (is anything maybe unclear or think about anything that would cause people to reach out to support)
 * [ ] Security review done [see checklist](https://developer.matomo.org/guides/security-in-piwik#checklist)
+* [ ] Wording review done
```

Comment:
Would it help to link to the developer docs here?

Reply:
added it 👍